parts/k8s/kubernetesconfigscustomcloud.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function ensureCertificates()
```

Comment:
nit: let's use the `ensureCertificates() {` function definition pattern (like you have below for `configureK8sCustomCloud`)